packages/ember/addon/index.ts

```diff
@@ -17,7 +17,9 @@ export function InitSentryForEmber(_runtimeConfig: BrowserOptions | undefined) {
   assert('Missing configuration.', config);
   assert('Missing configuration for Sentry.', config.sentry || _runtimeConfig);
 
-  const initConfig = Object.assign({}, config.sentry, _runtimeConfig || {});
+  // Permanently merge options into config, preferring runtime config
+  Object.assign(config.sentry, _runtimeConfig || {});
+  const initConfig = Object.assign({}, config.sentry);
```

Comment:
The only thing that `initConfig` does now, is shallow clone of `config.sentry`. And because the only thing you modify is an object (which is assigned by reference), it's effectively 2 objects with the same values (+ `_metadata` in case of `initConfig`). I wonder if it wouldn't be simpler to just go directly with `config.sentry` passed to `Sentry.init` instead. If you think not, just ignore the comment.